lib/readline.js

```diff
@@ -190,6 +190,12 @@ function Interface(input, output, completer, terminal) {
     this._ttyWrite = _ttyWriteDumb.bind(this);
   }
 
+  function onerror(err) {
+    const debug = require('internal/util/debuglog').debuglog('stream');
+    debug(err);
```

Comment:
Is this actually necessary? The error should be received due to emitting it right afterwards.